context-free.md

```diff
@@ -1,22 +1,25 @@
 # Context Free
 
 ##  Who is this programming language for?
-
+People who want to create digital art. 
```

Comment:
While this is true, there is a certain group of people it is really targeted at. Who are they? Answering questions about the target audience make analyzing a DSL much easier.